user/running-build-in-debug-mode.md

```diff
@@ -28,8 +28,12 @@ For public repositories, an API call is required, in addition to the feature bei
 
 ### Restarting a job in debug mode via API
 
-To restart a job in debug mode via API, send a `POST` request to the job's `debug` endpoint,
-along with your [Travis CI API token](/user/triggering-builds/) in the `Authorization` header.
+To restart a job in debug mode via API, send a `POST` request to the job's `debug` endpoint.
+This request needs to be authenticated by adding your [Travis CI API token](/user/triggering-builds/)
+to the `Authorization` header. You can find your API token in your Travis CI Profile page
+for [public projects](https://travis-ci.org/profile) or [private projects](https://travis-ci.com/profile),
+depending on the visibility of your repository.
```

Comment:
we don't mention 'depending on the visibility of your repository' anywhere else, it might confuse matters. WDYT?

Reply:
Oh - just saw this. I'm going to remove it, thanks :)